src/pages/taesooKim/Login/Login.scss

```
@@ -1 +1,79 @@
-// login
+/* Global */
+:root {
+  /* Color */
+  --color-white: #ffffff;
+  --color-light-white: #fafafa;
+  --color-dark-white: #dbdbdb;
+  --color-light-blue: #b2dffc;
+  --color-regular-blue: #0095f6;
+  --color-dark-blue: #4f7599;
+}
+
+.login {
+  position: relative;
+  width: 400px;
+  top: 100px;
+  margin: 0 auto;
+  padding-bottom: 20px;
+  border: 1px solid var(--color-dark-white);
+  background-color: var(--color-white);
+  .title {
+    display: flex;
+    justify-content: center;
+    margin: 40px 0 50px 0;
+    font-size: 50px;
+    font-family: "Lobster";
+  }
+  .login-container {
+    margin-top: 50px;
+    #login-form {
+      input {
+        display: block;
+        width: 300px;
```

```suggestion
  background-color: var(--color-white);
  
  .title {
    display: flex;
    justify-content: center;
    margin: 40px 0 50px;
    font-size: 50px;
    font-family: "Lobster";
  }
  
  .login-container {
    margin-top: 50px;
    
    .login-form {
      input {
        display: block;
        width: 300px;
```
- 새로운 셀렉터 앞에서 한줄 띄우기는 가독성 측면에서 기본적으로 적용해주셨으면 좋겠습니다. 링크 참고해보세요! https://www.notion.so/wecode/Westagram-JS-Refactoring-Checklist-dcff20de07b6449d89d82980e7af9cba#f909425e186044db85a827444dec61a9
- margin 단축 속성 사용가능합니다
- id 셀렉터는 정말 앱 전체에서 무조건 유일무이한 경우가 확실하지 않다면 쓸 필요가 크게 없을 거 같습니다

---

네네 수정하였습니다🐸